lib/maropost/contact.rb

```diff
@@ -1,39 +1,56 @@
 module Maropost
   class Contact
-    attr_accessor :id, :email,
-                  :ama_rewards, :ama_membership, :ama_insurance, :ama_travel, :ama_new_member_series, :ama_fleet_safety,
-                  :ovrr_personal, :ovrr_business, :ovrr_associate,
-                  :ama_vr_reminder, :ama_vr_reminder_email, :ama_vr_reminder_sms, :ama_vr_reminder_autocall,
-                  :phone_number, :cell_phone_number, :errors
+    ATTRIBUTES = %i(
+      id
+      email
+      phone_number
+      cell_phone_number
+      lists
+      errors
+    ).freeze
+    LISTS = %i(
```

Comment:
I like this. It makes it clearer what's a subscription vs a subscriber attribute. Too bad cell_phone_number is an outlier.